trainer_requirements.txt

```diff
@@ -0,0 +1,16 @@
+pygame
```

Comment:
We ideally shouldn't have a separate a trainer requirement's, otherwise it will get outdated fast (already is). Worst-case scenario, you should use both the normal `requirements.txt` and `trainer_requirements.txt`  with `trainer_requirements.txt` only containing dependencies exclusively used for training.

Reply:
The thing is, requirements.txt doesn't contain all of the packages needed for training. What I can do is copy trainer_requirements into requirements and download that?

Reply:
As I said use both `requirements.txt` and `trainer_requirements.txt`.

Reply:
E.g.

```
pip3 install -r $HOME/projects/PathBench/requirements.txt
pip3 install -r $HOME/projects/PathBench/trainer_requirements.txt
```